playbooks/roles/ocp-config/tasks/squid.yaml

```diff
@@ -0,0 +1,30 @@
+---
+
+- name: Install Squid package
+  package:
+    name: squid
+    state: present
+
+- name: Configure Squid
+  template:
+    src: etc/squid/squid.conf.j2
+    dest: /etc/squid/squid.conf
+    owner: root
+    group: root
+    mode: u=rw,g=r,o=r
+  become: true
+  notify:
+    - restart squid
+
+- name: Add Squid to firewall
+  firewalld:
+    service: squid
+    permanent: true
+    state: enabled
+    zone: public
+  notify:
+    - restart firewalld
+
+- name: Set proxy details
+  set_fact:
+    internal_proxy_url : "http://{{ ansible_host }}:{{ squid_http_port }}"
```

Comment:
should this be renamed to squid_proxy_url? or the setup_squid_proxy can be changed to setup_internal_proxy?  This will help to keep the variable names in sync

Reply:
I will rename it to squid_proxy_url. I have used name squid at many places including the file names.